docs/SnapshotTesting.md

```diff
@@ -128,13 +128,18 @@ Failed snapshots can also be updated interactively in watch mode:
 ![](/jest/img/content/interactiveSnapshot.png)
 
 Once you enter Interactive Snapshot Mode, Jest will step you through the failed
-snapshots one test suite at a time and give you the opportunity to review the
-failed output.
+snapshots one test at a time and give you the opportunity to review the failed
+output.
 
 From here you can choose to update that snapshot or skip to the next:
 
 ![](/jest/img/content/interactiveSnapshotUpdate.gif)
```

Comment:
in the new gif, mind turning off the text effect (https://github.com/zeit/hyper-power)? a bit distracting.

And maybe consider having something like https://github.com/keycastr/keycastr? I find it a bit confusing following what's going on 🙂 

Other than that, slick demo! 👍

Reply:
Updated, thanks for the tip